### ISSUE TITLE
Consider using var

### ISSUE THREAD

**Issue by sethladd** [andresaraujo/flutter_news#11]
When the variable is initialized immediately with a clear type, consider using var. `var` in Dart+strong mode (the default in Flutter) means "infer", so you don't lose type information.

Instead of:

```
  FlutterNewsConfiguration _configuration = new FlutterNewsConfiguration(
    themeName: ThemeName.light,
    showFullComment: false,
    expandCommentTree: false,
  );
```

You can do:

```
  var_configuration = new FlutterNewsConfiguration(
    themeName: ThemeName.light,
    showFullComment: false,
    expandCommentTree: false,
  );
```

**Comment by andresaraujo** [comment]
Thanks Seth, yeah and its easier to read

**Comment by andresaraujo** [comment]
Fixed in 1bdb9fcfc4b332feba75ff3e3a4246ca28e3605a
